control-plane/stor-port/src/types/v0/transport/replica.rs

```diff
@@ -441,6 +441,8 @@ pub struct CreateReplica {
     pub allowed_hosts: Vec<HostNqn>,
     /// The replica kind, eg: regular or clone.
     pub kind: Option<ReplicaKind>,
+    /// Data encryption.
+    pub encrypted: bool,
```

Comment:
Does this need option as well?

```suggestion
    pub encrypted: Option<bool>,
```

Reply:
Not sure, any advantage? It should just be true or false, None is indicative of false only.

Reply:
Option would let us know more information, ex: replica coming from node which doesn't know about encryption.